tests/PKSim.Tests/IntegrationTests/SimulationSpecs.cs

```diff
@@ -159,6 +166,54 @@ public void should_be_able_to_simulate_the_simulation()
       }
    }
 
+   public class When_creating_a_population_simulation_with_the_standard_building_block_and_iv_bolus_Asian_Tanaka_1996 : When_creating_a_population_simulation_with_the_standard_building_block_and_iv_bolus
```

Comment:
hum. Why not just iterate over all population instead of creating a test for each population explicitely? SImply use the PopulationRepositgory no?

Reply:
Just because I wanted to have an explicit message for every pop which fails.
Well could also iterate over the pops and collect error messages

Reply:
>Well could also iterate over the pops and collect error messages
Yes that would be the same. The advantage is that it would work for any other population automatically without having to write code.

If you want to leave it like that, this is also ok.